!Command
positional: []
named: []
command:
- svist4get
parent:
subcommands: []
help_flag:
usage_flag:
version_flag:
help_text: "Traceback (most recent call last):\n  File \"/tmp/tmprxeyeba8/bin/svist4get\"\
  , line 17, in <module>\n    parameters.parse_command_line()\n  File \"/tmp/tmprxeyeba8/lib/python3.7/site-packages/svist4get/manager.py\"\
  , line 33, in parse_command_line\n    with open(path_to_help, 'r') as short_help:\n\
  FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmprxeyeba8/lib/python3.7/site-packages/svist4get/svist4get_data/help/short_help.txt'\n\
  \nDuring handling of the above exception, another exception occurred:\n\nTraceback\
  \ (most recent call last):\n  File \"/tmp/tmprxeyeba8/bin/svist4get\", line 153,\
  \ in <module>\n    if parameters.cmd['debug']:\nAttributeError: 'Parameters' object\
  \ has no attribute 'cmd'\n"
generated_using:
- --help
